### PR TITLE
New presence, named Ro the Bot

### PR DESCRIPTION
**Pull request by bakzkndd**
I created a presence for the discord bot: Ro the Bot. This works on both the normal website and the status site. 

proof of functioning:

Schermopname (48)

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "name": "bakzkndd",
+        "id": "343103632453009408"
+    },
+    "service": "Ro the Bot",
+    "description": {
+        "en": "Ro the bot is the most diverse discord bot! Just imagine a function, and it probably has it."
+    },
+    "url": ["ro-the-bot.ga", "status.ro-the-bot.ga"],
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/avatars/547090981279891486/10fc0b186f58583d466672ac14027cb2.png?size=2048",
```
Comment (Alanexei) [inline metadata.json]: Please don't use Discord's CDN, use something like imgur or is-inside.me

Comment (bakzkndd) [reply metadata.json]: Changed it to an imgur

File: Ro the Bot/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "id": "343103632453009408"
+    },
+    "service": "Ro the Bot",
+    "description": {
+        "en": "Ro the bot is the most diverse discord bot! Just imagine a function, and it probably has it."
+    },
+    "url": ["ro-the-bot.ga", "status.ro-the-bot.ga"],
+    "version": "1.0",
+    "logo": "https://m.imgur.com/gallery/YDmTRVG",
+    "thumbnail": "https://rothebot.is-inside.me/8gHKV9oG.png",
```
Comment (Bas950) [inline Ro the Bot/dist/metadata.json]: For logo it should be an i.imgur url with the ending .png / .jpg
and upload the thumbnail picture also to imgur

Comment (bakzkndd) [reply Ro the Bot/dist/metadata.json]: made it i.imgur for both

**Pull request merged**